charts/temporal/templates/admintools-deployment.yaml

```diff
@@ -21,16 +16,8 @@ spec:
   template:
     metadata:
       labels:
-        app.kubernetes.io/name: {{ include "temporal.name" . }}
-        helm.sh/chart: {{ include "temporal.chart" . }}
-        app.kubernetes.io/managed-by: {{ .Release.Service }}
-        app.kubernetes.io/instance: {{ .Release.Name }}
-        app.kubernetes.io/version: {{ .Chart.AppVersion | replace "+" "_" }}
+        {{ include "temporal.resourceLabels" . | nindent 8 }}
         app.kubernetes.io/component: admintools
-        app.kubernetes.io/part-of: {{ .Chart.Name }}
-        {{- with $.Values.admintools.podLabels }}
```

Comment:
This functionality needs to stay. I suggest passing scope and resource type arguments to the helper, which will then check `.Values.${scope}.additionalLabels` and `.Values.${scope}.${type}Labels` alongside the current `.Values.addtionalLabels`.

Reply:
addressed @robholland . PTAL.